code/ai/ai_profiles.cpp

```diff
@@ -770,6 +772,10 @@ void ai_profile_t::reset()
 	if (mod_supports_version(3, 6, 10)) {
 		flags.set(AI::Profile_Flags::Reset_last_hit_target_time_for_player_hits);
 	}
+	// backwards compatible flag for a bug accidentally introduced during this time
+	if (mod_supports_version(3, 6, 14) && !mod_supports_version(23, 0, 0)) {
```

Comment:
If this PR is not going in until after stable, then this check should be `mod_supports_version(23, 0, 1)`

Reply:
Not 23.2 for the next stable?

Reply:
I would assume mods would want to test using nightly builds, which would be versioned 23.1.  You'd want the expression to work as soon as it is merged, not wait until we enter a new stable build cycle.

(Maybe 23.1 is a better fit than 23.0.1, but either way it shouldn't be 23.2.)

Reply:
Ok, I see. Nightlies seem to immediately increment the minor so I think 23.1 would do it.